frontend/src/components/client/controls/ClientModal.tsx

```diff
@@ -88,11 +89,12 @@ class ClientModalComponent extends Component<ClientModalProps, ClientModalState>
       </>
     );
 
+    const modalTitle = this.props.title ? this.props.title : t('client.createNewModal.client')
```

Comment:
In JavaScript kan je volgende doen

Relatief nieuw:

```
const modalTitle = this.props.title ?? t('client.createNewModal.client');
```

Dat gaat de t() nemen wanneer de props.title undefined is.

Er is ook:

```
const modalTitle = this.props.title || t('client.createNewModal.client');
```

Dat zal de t() nemen wanneer props.title "falsey" is (vb: null, undefined of lege string)